inmanta.spec

```diff
@@ -4,13 +4,15 @@
 %define buildid %{nil}
 %define buildid_egg %{nil}
 %define venv %{buildroot}/opt/inmanta
-%define _p3 %{venv}/bin/python3
+%define _p3 %{venv}/bin/python%{python_version}
 %define _unique_build_ids 0
 %define _debugsource_packages 0
 %define _debuginfo_subpackages 0
 %define _enable_debug_packages 0
 %define debug_package %{nil}
 
+%define python_version 3.6
```

Comment:
Does this work in this order?

Reply:
It does. But I am going to move it up to improve the readability of the file.